gr-blocks/lib/message_debug_impl.h

```diff
@@ -63,11 +65,12 @@ class message_debug_impl : public message_debug
     std::vector<pmt::pmt_t> d_messages;
 
 public:
-    message_debug_impl();
+    message_debug_impl(bool en_uvec);
     ~message_debug_impl();
 
     int num_messages();
     pmt::pmt_t get_message(int i);
+    void print_pdu_vector(bool en) { d_en_uvec = en; };
```

Comment:
Oh, that's a setter function? Is there a precedent in the code base for naming a setter function like this?